stdlib/public/core/StringGuts.swift

```diff
@@ -966,18 +966,22 @@ extension _StringGuts {
       }
     }
 
-    // TODO (TODO: JIRA): check if we're small and still within capacity
+    // Small strings can accomodate small capacities
```

Comment:
Note to self: use a spell checker. "accommodate"

Reply:
It allocc

but it also accommoddatt